### ISSUE TITLE
When creating a unittest with Runes $derived runes are  marked as "never reassigned"

### ISSUE THREAD

**Issue by mrh1997** [sveltejs/eslint-plugin-svelte#846]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

8.57.0

### What version of `eslint-plugin-svelte` are you using?

2.43.0

### What did you do?

```svelte
test('Counter', () => {
	let count = $state(0);
	let double = $derived(count * 2);

	count = 5;

	expect(double).toEqual(10);
});
```

### What did you expect to happen?

I expect that no error / warning is shown. Especially it should be detected that "double" will change its value.

### What actually happened?

At the "double" definition the following error is displayed:

`ESLint: 'double' is never reassigned. Use 'const' instead.(prefer-const`

**Comment by ota-meshi** [comment]
Duplicate of #818
